mezzanine/core/management/commands/createdb.py

```diff
@@ -26,25 +23,16 @@ class Command(BaseCommand):
     help = "Performs initial Mezzanine database setup."
     can_import_settings = True
 
-    def __init__(self, *args, **kwargs):
-        """
-        Adds extra command options (executed only by Django <= 1.7).
-        """
-        super(Command, self).__init__(*args, **kwargs)
-        if VERSION[0] == 1 and VERSION[1] <= 7:
-            self.option_list = migrate.Command.option_list + (
-                make_option("--nodata", action="store_true", dest="nodata",
-                    default=False, help="Do not add demo data."),)
-
```

Comment:
This copies the arguments from the `migrate` command, which the (existing) add_arguments() method doesn't. This has been fine for the last few versions so I'm assuming it's unnecessary.